hastexo/hastexo.py

```diff
@@ -110,11 +110,27 @@ class HastexoXBlock(XBlock,
         help="Set the progress check button label. "
              "For example: \"Submit Answer\" or \"Check Progress\"(Default)."
     )
+    show_feedback = Boolean(
+        default=True,
+        scope=Scope.settings,
+        help="On progress check, show feedback on how many tasks out of total "
+             "are completed. "
```

Comment:
Micro-nit: is this trailing whitespace after "completed." intentional?

Reply:
Not intentional, deleted now.